src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -163,6 +167,18 @@ public static FlowExecutionList get() {
         return l;
     }
 
+    /**
+     * Returns true if all executions that were present in this {@link FlowExecutionList} have been loaded and resumed.
```

Comment:
Is that last part actually true? Now that `onResume` can be called asynchronously, this does not seem to be tracking that it completed.

Reply:
The `WorkflowRun` and `CpsFlowExecution` have been loaded, `CpsFlowExecution.loadProgramAsync`, and `FlowExecutionListener.onResumed` have been called, but `StepExecution.onResume` may not have been called yet or may still be in progress.